tests/conftest.py

```diff
@@ -0,0 +1,53 @@
+import os
+
+import pytest
+import responses
+
+from mollie.api.client import Client
+
+
+@pytest.fixture(scope='session')
+def client():
+    """Setup a Mollie API client object."""
+    api_key = os.environ.get('MOLLIE_API_KEY', 'test_test')
+
+    client = Client()
+    client.set_api_key(api_key)
+    return client
+
+
+class ImprovedRequestsMock(responses.RequestsMock):
+    """Wrapper adding a few shorthands to responses.RequestMock."""
+
+    def get(self, url, filename, status=200):
+        """Setup a mock response for a GET request."""
+        body = self._get_body(filename)
+        self.add(responses.GET, url, body=body, status=status)
+
+    def post(self, url, filename, status=200):
+        """Setup a mock response for a POST request."""
+        body = self._get_body(filename)
+        self.add(responses.POST, url, body=body, status=status)
+
+    def delete(self, url, filename, status=204):
+        """Setup a mock response for a DELETE request."""
+        body = self._get_body(filename)
+        self.add(responses.DELETE, url, body=body, status=status)
+
+    def patch(self, url, filename, status=200):
+        """Setup a mock response for a PATCH request."""
+        body = self._get_body(filename)
+        self.add(responses.PATCH, url, body=body, status=status)
+
+    def _get_body(self, filename):
+        """Read the response fixture file and return it."""
+        # TODO refactor
```

Comment:
The idea was to use `pkg_resources` for this, but  since the tests are not part of the mollie package, that is not possible (just tried). Dropped the comment though.